homeassistant/components/light/iglo.py

```diff
@@ -0,0 +1,126 @@
+"""
+Support for lights under the iGlo brand.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/light.iglo/
+"""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.const import (CONF_HOST, CONF_NAME, CONF_PORT)
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS, ATTR_RGB_COLOR, ATTR_COLOR_TEMP,
+    SUPPORT_BRIGHTNESS, SUPPORT_COLOR_TEMP, SUPPORT_RGB_COLOR,
+    Light, PLATFORM_SCHEMA
+)
+
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['iglo==1.0.0']
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_NAME = 'iGlo Light'
+DEFAULT_PORT = 8080
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_HOST): cv.string,
+    vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.string,
+})
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Set up the iGlo lighs."""
+    host = config.get(CONF_HOST)
+    name = config.get(CONF_NAME)
+    port = config.get(CONF_PORT)
+    add_devices([IGloLamp(name, host, port)])
```

Comment:
Use [`True`](True) as second parameter.

I assume that there will be a exception of the device is not present.